components/microsoft_outlook/microsoft_outlook.app.mjs

```diff
@@ -138,13 +150,34 @@ export default {
     },
     prepareMessageBody(self) {
       const toRecipients = [];
+      const ccRecipients = [];
+      const bccRecipients = [];
       for (const address of self.recipients) {
         toRecipients.push({
           emailAddress: {
             address,
           },
         });
       }
+      for (const address of self.ccRecipients) {
+        if (address.trim() !== "") {
```

Comment:
The reasoning for the trim logic is because if I were to add then remove an item from the string array inputs, it seems it would still send an empty input instead of nothing. Because of this, the outlook API would get an array with an empty string in it which it interprets as an invalid recipient. To remedy this, I thought it would make sense to validate that there are no empty emails being put into the recipients lists as the cc and bcc lists should be optional.